doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1,6578 +1,16 @@
 :orphan:
 
 .. _known_issues:
+.. _known_issues_nrf5340audio:
+.. _ncsdk_19865:
+.. _drgn_15852:
+.. _krknwk_10611:
+.. _known_issues_other:
+.. _tnsw_46156:
+.. _ncsidb_925:
+.. _known_issue_tfm:
 
 Known issues
 ############
 
-.. contents::
```

Comment:
given https://github.com/nrfconnect/sdk-nrf/pull/24299#issuecomment-3290813752 I think we should add the following:

```
Other issues
************
.. rst-class:: v3-1-1

NRFX-8345: Idle current consumption might be increased by 8 uA in some scenarios.

  **Affected platforms:** nRF54H20

  **Workaround:** Currently unknown.
```